docs/source/about/contract.rst

```diff
@@ -44,7 +44,7 @@ HTTP/TCP services
 -----------------
 
 * MUST be discoverable by SmartStack
-* MUST bind to port 8888
+* MUST bind to port 8888 if using the ``'bridge'`` `networking mode <../yelpsoa_configs.html#net>`_ or to port $MARATHON_PORT if using the ``'host'`` networking mode
 
```

Comment:
This deserves and extra warning

* MUST NOT bind on any other ports than ones specified by `$MARATHON_PORT*` in host mode otherwise they will conflict with other services